.travis.yml

```diff
@@ -0,0 +1,17 @@
+dist: trusty
+language: node_js
+node_js:
+  - "8"
+cache:
+  directories:
+    - node_modules
+before_install:
+  - npm install -g greenkeeper-lockfile@1
+install: npm install
+before_script:
+  - greenkeeper-lockfile-update
+script:
```

Comment:
Perhaps we should add the `npm run build` command here as well?

So that we can make sure that both the `linting` works and that the website can be built without any build errors. So the linting command would run first and then the website will build second.

Reply:
Yeap, totally agree. Let's run the linter, tests and make sure is building correctly.

Reply:
Yeah I forgot about adding the `tests` command as well. :smile: 

I agree with that order. `linting`, `tests` and then `build`.

Reply:
> I'm ok with TravisCI, @jagzviruz can you add to our tech stack document why Travis over CircleCI?
> 
> https://docs.google.com/document/d/1DbdRZBsyjbeBl9vddUyXCUWsgwdgyyUY2NIQitf3_Z0/edit#heading=h.e0bl2e8hwksl
> 
> Thanks!

Will update it later in my evening.

Reply:
> Perhaps we should add the `npm run build` command here as well?
> 
> So that we can make sure that both the `linting` works and that the website can be built without any build errors. So the linting command would run first and then the website will build second.

If we're doing npm build just for the sake of linting, then the eslint loader of webpack will anyway be run the lint before the build.. so will be redundant acc to me

Reply:
Yup, also `lint-staged` is running before every commit. So, is it like 3 times linting will happen?